game.py

```diff
@@ -112,10 +172,12 @@ def canBuySettlement(self, resources):
             and resources['Wool'] >= 1 and resources['Wheat'] >= 1
 
     def canBuyDevCard(self, resources):
-        if not self.devCards: return False
+        if not self.devCards:
```

Comment:
Why are you putting this check here? should having a dev card have an effect on whether you can buy one?

Reply:
Good question where tf did that come from

Reply:
Oh wait...this is the game class and devCards are stored in a queue so this check makes sure the queue isn't empty and there are still devCards left to buy

Reply:
ah got ya, makes sense